core/domain/state_domain.py

```diff
@@ -2695,7 +2697,9 @@ def convert_state_dict_to_yaml(cls, state_dict, width):
                 'Bad state dict: %s' % python_utils.UNICODE(state_dict))
             python_utils.reraise_exception()
 
-        return python_utils.yaml_from_dict(state.to_dict(), width=width)
+        return yaml.dump(
+            state.to_dict(), allow_unicode=True,
+            default_flow_style=False, width=width)
```

Comment:
okay done now had missed it

Reply:
@vojtechjelinek is it necessary to move `yaml_from_dict`  to utils.py ? there are so few files using it.

@DiptoChakrabarty please remove `default_flow_style` flag where it is `false`, since it is the default

Reply:
okay

Reply:
Done

Reply:
I think it would be actually useful to move it to utils.py, also I just realized that you didn't even remove the `yaml_from_dict` from the _python_utils.py_ in this PR.

So what you should do is:
1. Move `yaml_from_dict` from the _python_utils.py_ to _utils.py_
2. Remove the `_recursively_convert_to_str` from `yaml_from_dict` and also remove `yaml_from_dict` from the _python_utils.py_
3. In all places that you've modified call the `yaml_from_dict`.

Reply:
Done